pkg/trivyk8s/trivyk8s_test.go

```diff
@@ -1,12 +1,224 @@
 package trivyk8s
 
 import (
+	"context"
+	"fmt"
 	"testing"
 
-	"github.com/aquasecurity/trivy-kubernetes/pkg/artifacts"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+
 	"github.com/stretchr/testify/assert"
+
+	"github.com/aquasecurity/trivy-kubernetes/pkg/artifacts"
+	"github.com/aquasecurity/trivy-kubernetes/pkg/bom"
+	"github.com/aquasecurity/trivy-kubernetes/pkg/k8s"
+	"github.com/aquasecurity/trivy-kubernetes/pkg/k8s/docker"
 )
 
+type MockClusterDynamicClient struct {
+	resource dynamic.NamespaceableResourceInterface
+}
+
+func (m MockClusterDynamicClient) Resource(schema.GroupVersionResource) dynamic.NamespaceableResourceInterface {
+	return m.resource
+
+}
+
+type MockNamespaceableResourceInterface struct {
+	err        error
+	namespaces []string
+}
+
+func (m MockNamespaceableResourceInterface) Namespace(s string) dynamic.ResourceInterface {
+	panic("implement me")
+}
```

Comment:
We can leave these functions as stubs 
```suggestion
func (m MockNamespaceableResourceInterface) Namespace(s string) dynamic.ResourceInterface {
	return <something>
}
```

Reply:
done https://github.com/aquasecurity/trivy-kubernetes/pull/395/commits/383660275b01f56037ea83bd5b8d4564ddd952d0